loaders/src/glTF/2.0/babylon.glTFLoaderInterfaces.ts

```diff
@@ -284,7 +284,7 @@ module BABYLON.GLTF2 {
         texCoord?: number;
     }
 
-    export interface IGLTF extends IGLTFProperty {
+    export interface _IGLTF extends IGLTFProperty {

```

Comment:
Why the _ ?

Reply:
This is an intermittent change so the glTF interface @bghgary is using does not stomp on the `babylon.glTFInterface.d.ts` declaration file in the same module namespace.

Reply:
Makes sense